roles/confluent.kafka_rest/tasks/main.yml

```diff
@@ -33,6 +33,14 @@
     recurse: yes
   when: sasl_protocol == 'kerberos'
 
+- name: Create Logs Directory
+  file:
+    path: "{{kafka_rest.appender_log_path}}"
+    owner: "{{kafka_rest.user}}"
+    group: "{{kafka_rest.group}}"
+    mode: '764'
```

Comment:
should we keep this task right above the create kafka rest log4j config task?
thats how it is in the other main.ymls